packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3095,3 +3096,36 @@ def get(self, id, authed_user_id):
         muted_users = get_muted_users(decoded_id)
         muted_users = list(map(extend_user, muted_users))
         return success_response(muted_users)
+
+
+email_key_response = make_response(
+    "email_key_response", ns, fields.String(required=False, allow_null=True)
+)
+
+
+@ns.route("/<string:id>/emails/key")
+class UserEmailKey(Resource):
+    @record_metrics
+    @ns.doc(
+        id="""Get User Email Key""",
+        description="Gets the email encryption key for a user by their user ID",
```

Comment:
Not sure if this would be the right place for it, but it might be nice to have more clarity about these things - what is an "email encryption key"? Maybe something like "Gets a key that can be used to decrypt emails that have been shared to the given user, which itself is encrypted and only usable by them."

Idk could use wordsmithing